src/geometry/ArborX_Ray.hpp

```diff
@@ -292,8 +297,8 @@ KOKKOS_INLINE_FUNCTION bool rayEdgeIntersect(Point const &edge_vertex_1,
 // when the ray and the triangle is coplanar.
 // In the paper, they just need the boolean return.
 KOKKOS_INLINE_FUNCTION
-bool intersection(Ray const &ray, Triangle const &triangle, float &tmin,
-                  float &tmax)
+bool intersection(Ray const &ray, Triangle<3, float> const &triangle,
```

Comment:
Don't need to specify `, float>` as that's the default.

Reply:
Doesn't hurt either for clarity.